main.mjs

```diff
@@ -0,0 +1,108 @@
+import { Run_Stage_Required_Scripts } from "./src/scripts/game.mjs";
+import { Previous_Stage } from "./src/scripts/stages.mjs";
+import { Next_Stage } from "./src/scripts/stages.mjs";
+
+window.onload = Run_Stage_Required_Scripts();
+const start = document.getElementsByClassName("btn");
+start[0].addEventListener("click", () => {

```

Comment:
Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/devkennyy/rungeon/pullRequest?prid=9807366)
```suggestion
start[0].addEventListener("click", () => {
```